rmw_implementation/package.xml

```diff
@@ -20,11 +20,11 @@
   <build_depend>rmw_connext_cpp</build_depend>
   <build_depend>rmw_cyclonedds_cpp</build_depend>
   <build_depend>rmw_fastrtps_cpp</build_depend>
+  <build_depend>rmw_implementation_cmake</build_depend>
```

Comment:
Could it be a `buildtool_depend`?

Also, I think it needs the `build(tool)_export_depend` as well, because the extra file find package's `rmw_implementation_cmake`:

https://github.com/ros2/rmw_implementation/blob/afe5761bddc3ca47994957d72e5514c291393a85/rmw_implementation/rmw_implementation-extras.cmake.in#L17